### PR TITLE
[Django40_warning] RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path()

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- from django.conf.urls import include, url -> 

  if sys.version_info[0] < 3:
    from django.conf.urls import include, url
  else:
    from django.urls import include, re_path as url 

## How was this patch tested?

- https://docs.djangoproject.com/en/3.1/releases/2.0/#what-s-new-in-django-2-0

### CODE REVIEW

File: apps/about/src/about/urls.py [hunk 1]
```diff
@@ -15,9 +15,15 @@
...
 
-from django.conf.urls import url
+import sys
+
 from about import views as about_views
 
+if sys.version_info[0] < 3:
+  from django.conf.urls import url
+else:
+  from django.urls import re_path as url
```
Comment (romainr) [inline apps/about/src/about/urls.py:25]: Could we be "forward" looking? (i.e. making the Django 3 first as this is the future going forward and using the same name)
```
if sys.version_info[0] > 2:
  from django.urls import re_path
else:
  from django.conf.urls import url as ...
```

**Review by romainr — COMMENTED** [review]
Is the behavior of both function backward compatible? (i.e. using url_path instead of url working out of the box)

File: apps/about/src/about/urls.py (hunk 1 above)
Comment (agl29) [reply apps/about/src/about/urls.py]: Done.
And re_path() is not backward compatible (compatible till Django 2).

Comment (romainr) [reply apps/about/src/about/urls.py]: It does not answer the question, reading the docs did "The django.conf.urls.url() function from previous versions is now available as django.urls.re_path()."

**Review by romainr — APPROVED** [review]

File: apps/about/src/about/urls.py (hunk 1 above)
Comment (agl29) [reply apps/about/src/about/urls.py]: yes, that's why I put that link in the description (please see in How was this patch tested?) @romainr
some lines from the link ->
The django.conf.urls.url() function from previous versions is now available as django.urls.re_path(). The old location remains for backwards compatibility, without an imminent deprecation. The old django.conf.urls.include() function is now importable from django.urls so you can use from django.urls import include, path, re_path in your URLconfs.

**Pull request merged**